lib/dictBuilder/cover.c

```diff
@@ -276,11 +286,17 @@ static int COVER_cmp8(COVER_ctx_t *ctx, const void *lp, const void *rp) {
 
 /**
  * Same as COVER_cmp() except ties are broken by pointer value
- * NOTE: g_coverCtx must be set to call this function.  A global is required because
- * qsort doesn't take an opaque pointer.
  */
+#if defined(_WIN32) && defined(_MSC_VER)
```

Comment:
The way `COVER_strict_cmp()` (and below `COVER_strict_cmp8()`) are implemented "feels" unusual : they share the same (short) body but feature different signatures depending on target system, controlled through preprocessor branching.
 
Given that the function signature is a mandated parameter of the `qsort_r()` function, and `qsort_r()` (of course!) varies depending on target system, I don't have a better solution to propose. Rather, I just forward some minor comments : 
    - `WIN_CDECL` is used even in non-Windows context. Granted, it's reduced to nothing in non-Windows environments, but seeing it listed for systems specifically tested as non Windows feels irrelevant
    - `_MSC_VER` and `__APPLE__` environments seem to share the same signature, so _maybe_ they could be merged (?)

Ultimately, this is a comment focused on "readability". The functionality itself looks good.